.github/workflows/R-CMD-check.yaml

```diff
@@ -39,11 +39,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: r-lib/actions/setup-r@v1
+      - uses: r-lib/actions/setup-r@v2
         with:
           r-version: ${{ matrix.config.r }}
 
-      - uses: r-lib/actions/setup-pandoc@v1
+      - uses: r-lib/actions/setup-pandoc@v2
```

Comment:
```suggestion
```

Reply:
our vignettes now are being rendered with markdown package which does not need pandoc